src/docs/feature-launch-process.md

```diff
@@ -2,24 +2,76 @@
+
+### Fuzzing is required to ship { #fuzzing }
+
+JavaScript and WebAssembly features must be fuzzed for a minimum period of 4 weeks, or one (1) release milestone, with all fuzz bugs fixed, before they can be shipped.
+
+For code-complete JavaScript features, start fuzzing by moving the feature flag to the `JAVASCRIPT_STAGED_FEATURES_BASE` macro in [`src/flags/flag-definitions.h`](https://source.chromium.org/chromium/chromium/src/+/master:v8/src/flags/flag-definitions.h).
+
+For WebAssembly, see the [WebAssembly shipping checklist](/docs/wasm-shipping-checklist).
+
+### [Chromestatus](https://chromestatus.com/) and review gates { #chromestatus }
+
+The blink intent process includes a series of review gates that must be approved on the feature's entry in [Chromestatus](https://chromestatus.com/) before an Intent to Ship is sent out seeking API OWNER approvals.
+
+These gates are tailored towards web APIs, and some gates may not be applicable to JavaScript and WebAssembly features. The following is broad guidance. The specifics differ from feature to feature; do not apply guidance blindly!
+
+#### Privacy
 
-Adding Web Platform Tests (WPT) is not required, as JavaScript and WebAssembly language features have their own test repositories. Feel free to add some though, if you think it is beneficial.
+Most JavaScript and WebAssembly features do not affect privacy. Rarely, features may add new fingerprinting vectors that reveal information about a user's operating system or hardware.
 
-For JavaScript features, explicit correctness tests in [Test262](https://github.com/tc39/test262) are preferred and required.
+#### Security
+
+While JavaScript and WebAssembly are common attack vectors in security exploits, most new features do not add additional attack surface. [Fuzzing](#fuzzing) is required, and mitigates some of the risk.
+
+Features that affect known popular attack vectors, such as `ArrayBuffer`s in JavaScript, and features that might enable side-channel attacks, need extra scrutiny and must be reviewed.
+
+#### Enterprise
+
+Throughout their standardization process in TC39 and the Wasm CG, JavaScript and WebAssembly features already undergo heavy backwards compatibility scrutiny. It is exceedingly rare for features to be willfuly backwards incompatible.
+
+For JavaScript, recently shipped features can also be disabled via `chrome://flags/#disable-javascript-harmony-shipping`.
+
+#### Debuggability
+
+JavaScript and WebAssembly features' debuggability differs significantly from feature to feature. JavaScript features that only add new built-in methods do not need additional debugger support, while WebAssembly features that add new capabilities may need significant additional debugger support.
+
```

Comment:
I don't feel strongly about this as this is errata documentation, but would it be useful to surface what debuggability means for language features in more detail? The [Wasm feature debugging checklist](https://goo.gle/devtools-wasm-checklist), is also linked from the shipping checklist for Wasm features, but I don't see a link to the [JS language feature debugging](https://docs.google.com/document/d/1_DBgJ9eowJJwZYtY6HdiyrizzWzwXVkG5Kt8s3TccYE/edit#heading=h.u5lyedo73aa9) checklist linked anywhere else.

Reply:
Great call, will add links.